### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by axelson**
Also add a test to do basic verification of changelog links

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 ### Unreleased
 
+Improvements:
+- Use fuzzy matching for completion (thanks [Po Chen](https://github.com/princemaple)) [#491](https://github.com/elixir-lsp/elixir-ls/pull/491)
```
Comment (princemaple) [inline CHANGELOG.md:4]: function completion only, for now

Comment (axelson) [reply CHANGELOG.md]: Ah, yes. Good point. Adjusted the wording in https://github.com/elixir-lsp/elixir-ls/pull/499/commits/4cdbd248d8f35f1649f14579da30c269a58f51cf

**Comment by princemaple** [comment]
Is this intended for a release?

**Comment by axelson** [comment]
> Is this intended for a release?

Yes it is. I just want to do some local testing with #515. Planning to do a release this weekend or early next week.

**Comment by princemaple** [comment]
Great 🎉 I'm looking forward to the community feedbacks on the auto complete change, before adding it to more places.

**Pull request merged**